### DISCUSSION TITLE
Slack Notifications on Webhook Deployments

### DISCUSSION THREAD

**Discussion by BenGale** [vercel/community#498] [Idea]
Not a major issue just a bit of an annoyance really. I have slack notifications for deployments enabled, and it shows the commit title for the change that triggers the deployment, which is great. However, when I trigger a deployment with a webhook from my CMS changing, it also shows the last commit title, which is a little confusing for people since a git change didn't trigger the deployment. It would be great if it just showed a message saying it was triggered by the CMS.

**Comment by wiiiimm** [comment]
I have [forwarded](https://github.com/vercel/product/issues/386) your feedback to our team. Thanks for pointing this out to us.
